src/cca/app/scripts/cordova_browser_api.ts

```diff
@@ -50,14 +50,43 @@ class CordovaBrowserApi implements BrowserAPI {
   }
 
   public startUsingProxy = (endpoint:net.Endpoint) => {
-    // TODO: Implement getter support, possibly using "redsocks".
+    if (!chrome.proxy) {
```

Comment:
chrome.proxy is not available in Cordova yet.  I have an experimental implementation on my machine, but I haven't yet made it publicly available.  In fact, manifest.json currently doesn't even ask for "proxy" permission (which I expect would fail on Cordova), so as checked in, this method will always return early.